apps/.storybook/config.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```

Comment:
it's the `react-storybook` convention. We can customize it to something else if that would be preferable.